gulpfile.js

```diff
@@ -410,3 +409,8 @@ gulp.task('prod', function(callback) {
 gulp.task('dev:runserver', function(callback) {
     runSequence('dev', 'connect', 'watch', 'test:watch', callback);
 });
+// Syncserver for development in backend
+gulp.task('dev:syncserver', function(callback) {
```

Comment:
@viditjain08 What's the difference between dev:runserver and dev:syncserver.. both will do the same thing as you have changed the source directory. (except server running)

Reply:
Yes, but I thought it would be better if there are no servers running (frontend + phantomjs) when thr person wants to use only a single server, plus in heroku deployment, we wont be able to run multiple servers.

Reply:
So my question is that how will I start dev server then if I developing locally. It will also sync Backend server. Isn't it.
You should write different tasks for the syncserver. Correct me if I'm missing something please @viditjain08

Reply:
Different tasks can be written, but it would be pointless to maintain 2 directories (one for backend and other for frontend) since previously, there was only 1 which was watched, so indirectly, it was also synced with backend except for the fact that they were on different servers.
But if you want I can write different tasks for the syncserver.

Reply:
Read this, maybe it will help you to understand why we have accepted Frontend driven approach. https://developer.telerik.com/featured/front-end-driven-applications-new-approach-applications/ @viditjain08

Reply:
I have added a commit which maintains the frontend directory as before when gulp dev:runserver, and a different directory with syncserver. 
But I dont know why travis ci is not able to use the cache while checking.
What may be the problem?
@aka-jain

Reply:
Looks Good, thanks.